src/connection/socket-connection.ts

```diff
@@ -0,0 +1,208 @@
+import type { Dwn, GenericMessage } from "@tbd54566975/dwn-sdk-js";
+import { DwnMethodName } from "@tbd54566975/dwn-sdk-js";
+
+import type { WebSocket } from "ws";
+import log from 'loglevel';
+import { v4 as uuidv4 } from 'uuid';
+
+import type { RequestContext } from "../lib/json-rpc-router.js";
+import type { JsonRpcErrorResponse, JsonRpcId, JsonRpcRequest, JsonRpcResponse } from "../lib/json-rpc.js";
+
+import { requestCounter } from "../metrics.js";
+import { jsonRpcApi } from "../json-rpc-api.js";
+import { JsonRpcErrorCodes, createJsonRpcErrorResponse, createJsonRpcSuccessResponse } from "../lib/json-rpc.js";
+import { DwnServerError, DwnServerErrorCode } from "../dwn-error.js";
+
+const SOCKET_ISALIVE_SYMBOL = Symbol('isAlive');
```

Comment:
For my understanding (I know this was existing code): do you know what's the reason for creating a `Symbol` for this string? Would a dumb private `isAlive` boolean member in `SocketConnection` not work?

Reply:
Removed in favor of private `isAlive` member.